src/main/scala/cromwell/binding/Scope.scala

```diff
@@ -4,20 +4,21 @@ import cromwell.binding.AstTools.AstNodeName
 import cromwell.parser.WdlParser.{Ast, AstList}
 
 import scala.annotation.tailrec
-import scala.collection.mutable.{Map, HashMap}
 import scala.collection.JavaConverters._
+import scala.collection.mutable
 
 object Scope {
-
   /**
    * Generate a Scope Tree from an Ast.
```

Comment:
The Scaladoc seems more generic than the actual functionality if this is intended to only be invoked on workflow ASTs.  Should the code be making an assertion to this effect?